README.md

```diff
@@ -108,7 +108,7 @@ router.post('/webhooks/amio-communicator', async (req, res) => {
 
 3. Implement **event handlers**. 
 ```js 
-amioWebhookRouter.onMessageReceived((data, timestamp) => {
+amioWebhookRouter.onMessageReceived(async (data, timestamp) => {
```

Comment:
To fakt chces mit async? Jak to vypada v realne implementaci? Tam das spis nejaky handler v jinem file, ne?